Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.37692&attempt_id=d50dfb21-4a18-4407-ad4b-0ccb91eb7588)

Attempted: systemd

<details><summary>Partial log (click to expand)</summary><p>

```
[601/1461] Compiling C object 'systemd-coredump@exe/src_coredump_coredump.c.o'.
[602/1461] Compiling C object 'systemd-resolved@exe/src_resolve_resolved-dns-scope.c.o'.
[603/1461] Compiling C object 'systemd-resolved@exe/src_resolve_resolved-manager.c.o'.
[604/1461] Compiling C object 'loginctl@exe/src_login_loginctl.c.o'.
[605/1461] Compiling C object 'systemd-resolve@exe/src_resolve_resolve-tool.c.o'.
[606/1461] Compiling C object 'machinectl@exe/src_machine_machinectl.c.o'.
[607/1461] Compiling C object 'systemctl@exe/src_systemctl_systemctl.c.o'.
ninja: build stopped: subcommand failed.
builder for '/nix/store/0cdrggb66j4h3nmljzxbcsfhd2448kq0-systemd-238.drv' failed with exit code 1
[31;1merror:[0m build of '/nix/store/0cdrggb66j4h3nmljzxbcsfhd2448kq0-systemd-238.drv' failed
```
</p></details>

--- a/pkgs/os-specific/linux/systemd/default.nix
+++ b/pkgs/os-specific/linux/systemd/default.nix
@@ -21,11 +21,13 @@ in stdenv.mkDerivation rec {
   version = "238";
   name = "systemd-${version}";
 
+  # When updating, use https://github.com/systemd/systemd-stable tree, not the development one!
+  # Also fresh patches should be cherry-picked from that tree to our current one.
   src = fetchFromGitHub {
     owner = "NixOS";
     repo = "systemd";
-    rev = "243d65d38f2df82d4a39f6a9970337803dff65a1";
-    sha256 = "098hxlkh6q17rxa178adylksxnnd4x9rxb8amjnlbiydcc6kaa5n";
+    rev = "3a439bcdb5706dbb44215ef4e70f07b09aaac040";
+    sha256 = "0qkk5891068pkxmxqvm07bwl597i8lfp89c23yxp11m21cjq0f4b";
   };
 
   outputs = [ "out" "lib" "man" "dev" ];

